docs/managing-cpp-deps.md

```diff
@@ -0,0 +1,48 @@
+---
+id: managing-cpp-deps
+title: Managing C++ dependencies
+---
+
+Details to consider when consuming a Visual C++ project as a source dependency of your app.
+
+### Details
+
+Applications with native code, either written on `C#` or `C++`, may add source dependencies on native Visual C++ (`.vcxproj`) projects.\
+Meaning, such dependencies will be built as part of the application.
+
+Starting with version `0.68`, React Native for Windows apps use the [`PackageReference`](https://docs.microsoft.com/en-us/nuget/consume-packages/package-references-in-project-files) restore project style for native `C++` NuGet dependencies.\
+The main change consists in NuGet packages being directly loaded from the user account's `globalPackagesFolder` cache instead of also copying them into a local folder relative to the Visual Studio Solution location (see [`repositoryPath`](https://docs.microsoft.com/en-us/nuget/reference/nuget-config-file)).
+
+This may conflict with Node/Visual C++ dependencies that use the more common [`packages.config`](https://docs.microsoft.com/en-us/nuget/reference/packages-config) project style (i.e. [React Native Picker](https://github.com/react-native-picker/picker#react-native-pickerpicker)).
+
+## Updating your app
+
+### Add the React Native Windows C++ Dependency (i.e. RN Picker)
+
+```
+yarn add @react-native-picker/picker
+```
+
+### Add a `packages.config` file to your app's project directory
```

Comment:
Historically, autolinking would manage this file. Is that still the case?

Reply:
Er, I might be mixing this up. I remember there was some differences of NuGet restore behavior for transitive dependencies on packages.config that @jonthysell ran into before.

Reply:
Wait, is this just as simple as adding a packages.config to that app's project directory?

Is there any reason we're not just putting/modifying a file there ourselves during autolinking? Or just manually calling nuget restore on the lib project so the files get put in the right place?

Reply:
Calling NuGet restore doesn't work once PackageReference is detected.

I did not know we could have Autolink set this up for us.
We could add such functionality and then remove this instruction.